ios/Sources/Repository/Authenticator.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  Authenticator.swift
+//  
+//
+//  Created by watanavex on 2021/06/12.
+//
+
+import Foundation
+import DroidKaigiMPP
+import FirebaseAuth
+
+class AuthenticatorImpl: Authenticator {
+
+    func currentUser(completionHandler: @escaping (DroidKaigiMPP.User?, Error?) -> Void) {
+        guard let firebaseUser = Auth.auth().currentUser else {
+            completionHandler(nil, nil)
```

Comment:
I think this line should pass user or error. How about this?

Reply:
This is a successful state like below:
https://github.com/DroidKaigi/conference-app-2021/blob/4cbbbbedfbcf26ed86eb1ad7b576dcdeed4564db/data/api/src/main/java/io/github/droidkaigi/feeder/data/Authenticator.kt#L11-L12

Reply:
I understand. Kotlin's suspend function doesn't represent Error state but Swift interface has Error state.
So I felt it is unnatural 😢 

It's okey.